src/searchForm/SearchForm.tsx

```diff
@@ -248,7 +272,11 @@ export const SearchForm = ({
                   { 'kirk-search-placeholder': !autocompleteFromValue },
                 ])}
               >
-                {autocompleteFromValue?.item.label || autocompleteFromPlaceholder}
+                {getPlaceholderText(
```

Comment:
Can the `initial(From|To)` values be displayed in black as if the user selected it?
I didn't see it at first in the screenshot in the description because it had the same color as a placeholder.

Reply:
You're right, it's probably better to leave it "normal" in the absence of product feedback for now. 
I removed the placeholder class when an initial value is present.